src/layers/medPython/base/objects/medPythonModule.cpp

```diff
@@ -72,7 +72,14 @@ PyObject* Module::getModuleObject(QString name)
 
     if (!moduleObject)
     {
-        moduleObject = coreFunction(PyImport_ImportModuleLevel, qUtf8Printable(name), nullptr, nullptr, *list(), 0);
+        try
+        {
+            moduleObject = coreFunction(PyImport_ImportModuleLevel, qUtf8Printable(name), nullptr, nullptr, *list(), 0);
+        }
+        catch (med::python::BaseException &e)
+        {
+            qDebug() << e.what();
```

Comment:
I don't think the exception should be stopped inside this function because it is low-level. More generally alll code using the medPython library should be aware that the API may raise exceptions, and so it is the calling code that should make sure to catch them.
(I know we had a crash relating to an exception that wasn't handled during initialisation and I'll fix that)

Reply:
ok I will remove it

Reply:
done